src/main/java/calculator/PositiveNumber.java

```diff
@@ -14,4 +14,8 @@ public PositiveNumber(int value){
     public int getValue(){
         return value;
     }
+
+    public PositiveNumber sum(PositiveNumber number) {
+        return new PositiveNumber(value + number.getValue());
+    }
```

Comment:
```suggestion
    public PositiveNumber sum(PositiveNumber number) {
        return new PositiveNumber(value + number.value);
    }
```
- getter를 사용하지 않고 접근이 될거에요~